Graphics/Implicit/Export/NormedTriangleMeshFormats.hs

```diff
@@ -1,50 +1,43 @@
 -- Implicit CAD. Copyright (C) 2011, Christopher Olah (chris@colah.ca)
 -- Copyright (C) 2016 Julia Longtin (julial@turinglace.com)
 -- Released under the GNU AGPLV3+, see LICENSE
-
```

Comment:
hmm. this line is intentional, to seperate the license, from the language extensions.

Reply:
8b70a86 ?

Alternatively, you can just keep adding `--` lines until the distance looks good even without the completely empty lines.

Hm.

Reply:
I can be convinced of that.